.github/workflows/maven-release.yml

```diff
@@ -1,16 +1,20 @@
-name: Deploy to Maven
+name: Publish Release
```

Comment:
Maybe "Maven Release" to match the file name?

Reply:
I've renamed the file as it does more than Maven release and I want to reflect that.